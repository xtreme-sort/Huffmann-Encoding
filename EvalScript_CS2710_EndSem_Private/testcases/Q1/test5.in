BuildTreeFromFrequencyTable
9
V 38
g 40
2 42
b 8
M 10
F 45
S 32
a 14
0 19
Quit
