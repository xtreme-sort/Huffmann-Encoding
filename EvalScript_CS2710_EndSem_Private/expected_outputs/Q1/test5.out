((F,(0,S)),(((a,(b,M)),V),(g,2)));
